docs/release_notes/v0.11.0.md

```diff
@@ -0,0 +1,489 @@
+
+##### 1. Get the latest CLI
+
+Get the latest version of the Dapr CLI as outlined above, and put it in your path.
+You can also use the helper scripts outlined [here](https://github.com/dapr/cli#installing-dapr-cli) to get the latest version.
+
+##### 2. Upgrade
+
+First, update your Helm repos:
+
+```
+helm repo update
+```
+
+Run the following command to upgrade Dapr:
+
+```
+helm upgrade dapr dapr/dapr --version 0.11.0 --namespace dapr-system --reset-values
+```
+
+Wait until all the pods are in Running state:
+
+```
+kubectl get pods -w -n dapr-system
+```
+
+Verify the control plane is updated and healthy:
+
+```
+dapr status -k
+```
+
+Next, issue a rolling update to your Dapr enabled deployments. When the pods restart, the new sidecar version will be picked up.
+
+```
+kubectl rollout restart deploy/<DEPLOYMENT-NAME>
+```
+
+All done!
+
+#### Starting fresh
+
+If you previously installed Dapr on your Kubernetes cluster using the Dapr CLI, run:
+
+*Note: Make sure you're uninstalling with your existing CLI*
+
+```bash
+dapr uninstall --kubernetes
+```
+
+It's fine to ignore any errors that might show up.
+
+If you previously installed Dapr using __Helm 2.X__:
+
+```bash
+helm del --purge dapr
+```
+
+If you previously installed Dapr using __Helm 3.X__:
+
+```bash
+helm uninstall dapr -n dapr-system
+```
+
+Update the Dapr repo:
+
+```bash
+helm repo update
+```
+
+If you installed Dapr with Helm to a namespace other than `dapr-system`, modify the uninstall command above to account for that.
+
+You can now follow [these](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#using-helm-advanced) instructions on how to install Dapr using __Helm 3__.
+
+Alternatively, you can use the CLI:
+
+```
+dapr init --kubernetes
+```
+
+#### Post installation
+
+Verify the control plane pods are running and are healthy:
+
+```
+dapr status -k
+```
+
+After Dapr 0.11.0 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
+This can be done with:
+
+```
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
+
+### Dapr Runtime
+
+#### New ClusterRole for dapr-operator
+
+Removed Cluster Admin permissions and now the dapr-operator has more fine grained [permissions](https://github.com/dapr/dapr/issues/1997). 
+
+#### Serialization for gRPC State Transaction APIs have changed 
+
+Before v0.11.0, on calling the State Transaction API via gRPC, the runtime was converting byte arrays to string and calling the respective component to handle the transaction. 
+
+```go
+Value:    string(req.Value),
+```
+
+Now the runtime no longer coverts the byte array to string, and the individual component needs to handle the serialization properly. 
+
+```
+Value:    req.Value,
+```
+
+Read more [here](https://github.com/dapr/dapr/issues/2016).
+
+
+### Components Contrib 
+
+#### Serialization for State Transaction APIs have to be handled in component 
```

Comment:
Unless I'm missing something, this is not a breaking change as far as the end user is concerned and this entire section can be removed.

Reply:
This will be a breaking change for gRPC. Since the old code will not work, since the double serialization issue has been fixed in this release.

Reply:
We're talking about breaking changes for end-users, not developers. So what does it mean for an end user that the serialization "have to be handled in component"? Also, the entire piece of code represents internal Dapr workings which should not be of interest to users. IMO we can remove this entire block.

Reply:
The payload would have been changed. The API will break with old serialized data.

Reply:
/cc @mchmarny @msfussell can you assist in communicating this better to non-Dapr coding users?